BeVolt/Drivers/src/LTC6811.c

```diff
@@ -89,19 +267,18 @@ void LTC6811_Init(cell_asic *battMod){
 	GPIO_Init(GPIOB, &GPIO_InitStruct);
 	GPIO_SetBits(GPIOB, GPIO_Pin_6);
 	
-	LTC681x_init_cfg(NUM_VOLTAGE_BOARDS, battMod);
+	LTC6811_init_cfg(NUM_VOLTAGE_BOARDS, battMod);
 	LTC6811_reset_crc_count(NUM_VOLTAGE_BOARDS, battMod);
 	LTC6811_init_reg_limits(NUM_VOLTAGE_BOARDS, battMod);
 }
 
-/********************************************************
-*********************************************************/
-
-
-void LTC6811_init_reg_limits(uint8_t total_ic, cell_asic ic[])
-{
-  for (uint8_t cic=0; cic<total_ic; cic++)
-  {
+/* LTC6811_init_reg_limits
+ * Initializes variables for each voltage board
+ * @param unsigned int 8-bit Total number of IC
+ * @param cell_asic struct
+ */
+void LTC6811_init_reg_limits(uint8_t total_ic, cell_asic ic[]){
```

Comment:
@SijWoo Seeing these changes reminds me of another thing. We should add to the style guide bracket guidelines. Personally, I'm a fan of [linux kernel style](https://en.wikipedia.org/wiki/Indentation_style#Variant:_Linux_kernel) for the braces, but as long as we have a standard, it should make the code more readable.